gprofiler/merge.py

```diff
@@ -11,8 +11,8 @@
 from typing import Iterable, Optional, Tuple
 
 from gprofiler.docker_client import DockerClient
+from gprofiler.gprofiler_types import ProcessToStackSampleCounters, StackToSampleCount
```

Comment:
Why this rename?

Reply:
`types` messes up a lot of imports, it prevented me from debugging the code

Reply:
.